MEDO v0.5.owl

```diff
@@ -804,6 +975,536 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_formative_peer_assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_formative_peer_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000103"/>
+        <obo:IAO_0000115>A type of formal formative assessment where fellow learner(s) observes, assesses, and provides written or oral subjective feedback of a learner&apos;s work or performance.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM13: The concurrent or retrospective review by learners of the quality and efficiency of practices or services ordered or performed by fellow learners.
+
+Does not include: Multisource Assessment; Narrative Assessment</obo:IAO_0000119>
+        <rdfs:label>formal formative peer assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM13: Peer Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_formative_self-assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_formative_self-assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000103"/>
+        <obo:IAO_0000115>A type of formal formative assessment where student observes, evaluates, and reflects upon one’s own deficiencies, achievements, behavior or professional performance, and competencies. The assessment is completed by the learner to reflect upon and critically assess their own performance against a set of established criteria.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM17: The process of evaluating one’s own deficiencies, achievements, behavior or professional performance and competencies (Institute for International Medical Education, 2002); Assessment completed by the learner to reflect and critically assess his/her own performance against a set of established criteria (Gordon, 1991) (NOTE: Does not refer to NBME Self-Assessment)
+
+Synonymous with/Includes: Observation of self on video
+
+Does not include: NBME Self-Assessment (National Board of Medical Examiners (NBME) Self-Assessment Services, n.d.)</obo:IAO_0000119>
+        <rdfs:label>formal formative self-assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM17: Self-Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_formative_teacher_observation_and_feedback -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_formative_teacher_observation_and_feedback">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000103"/>
+        <obo:IAO_0000115>A type of formal formative assessment where teacher or observer(s) observes, assesses, and provides written or oral subjective feedback of a learner&apos;s work or performance.</obo:IAO_0000115>
```

Comment:
The Medbiquitous definition specifies only written feedback, not oral. Is it your intention to have a broader definition? If so, should there be two child classes, one for 'written formal formative teacher observation and feedback' and a second for 'oral formal formative teacher observation and feedback'? The Medbiquitous AM10 could then be linked to the written one. Also, I would write "one or more teachers or observers observe assess, and provide..."